charts/seed-controlplane/charts/kube-apiserver/templates/networkpolicy-allow-kube-apiserver-policy.yaml

```diff
@@ -42,7 +42,7 @@ spec:
           app: vpa-admission-controller
     ports:
     - protocol: TCP
-      port: 443
+      port: 10250
```

Comment:
Perhaps this port should be configurable via `values.yaml`. Also, looking at the description of this policy it seems wrong / insufficient, that's why I didn't recognize it should be changed. Perhaps the description should be updated as well?

Reply:
Yes, agreed. I didn't make it configurable yet because the Go code is not explicitly setting the port value when rendering the VPA chart, so I did it similarly here (to only have a minimal change to get the issue fixed).

Generally, I don't like that we are relying on the default values in the Helm charts on too many places (which is also the reason why it this issue appeared in the first place). Ideally, we define the constants/values in Go and pass them for rendering - everywhere.
This requires some time-consuming refactoring, though... but I really would like to change that. The testability as it's now is not very good.

Reply:
OK, thanks for the explanation!